ospfd/ospf_te.c

```diff
@@ -2311,6 +2311,12 @@ static int ospf_te_parse_te(struct ls_ted *ted, struct ospf_lsa *lsa)
 			edge->status = UPDATE;
 	}
 
+	if (!admin_group_cmp(&old->ext_admin_group, &attr.ext_admin_group)) {
+		admin_group_copy(&old->ext_admin_group, &attr.ext_admin_group);
+		if (edge->status != NEW)
+			edge->status = UPDATE;
+	}
+
```

Comment:
Hum, I think it is a residual modification for OSPF. Should be completed or addressed in another PR.

Reply:
This PR is about IS-IS. OSPF implementation should be added in another PR

Reply:
done